tests/unit/checkboxradio/checkboxradio_options.js

```diff
@@ -0,0 +1,180 @@
+-/*
+ * checkboxradio_methods.js
+ */
+
+
+(function($) {
+
+module( "Checkboxradio: checkbox: options" );
+	test( "options: disabled", function() {
+		var checkbox = $( "#checkbox-option-disabled" ),
+			widget;
+		expect( 10 );
+		checkbox.checkboxradio({
+			disabled: true
+		});
+
+		widget = checkbox.checkboxradio( "widget" );
+
+		strictEqual( widget.hasClass( "ui-state-disabled" ), true,
+			"label gets ui-state-disabled when initial option set to true" );
+		strictEqual( checkbox.is( ":disabled" ), true,
+			"checkbox is disabled when inital option is set to true" );
+
+		checkbox.checkboxradio( "option", "disabled", false );
+
+		strictEqual( widget.hasClass( "ui-state-disabled" ), false,
+			"label has ui-state-disabled removed when disabled set to false" );
+		strictEqual( checkbox.is( ":disabled" ), false,
+			"checkbox has disabled prop removed when disabled set to false" );
+
+		checkbox.checkboxradio( "option", "disabled", true );
+
+		strictEqual( widget.hasClass( "ui-state-disabled" ), true,
+			"label gets ui-state-disabled when option set to true" );
+		strictEqual( checkbox.is( ":disabled" ), true,
+			"checkbox is disabled when option is set to true" );
+
+		checkbox.checkboxradio( "destroy" );
```

Comment:
Should put this in a separate test, the next destroy-setup-assert block as well.